internal/appsec/dyngo/instrumentation/httpsec/http.go

```diff
@@ -105,28 +125,59 @@ func MakeHandlerOperationArgs(r *http.Request, pathParams map[string]string) Han
 
 // Operation type representing an HTTP operation. It must be created with
 // StartOperation() and finished with its Finish().
-type Operation struct {
-	dyngo.Operation
-	events json.RawMessage
-}
+type (
+	Operation struct {
+		dyngo.Operation
+		events json.RawMessage
+	}
+
+	// SDKBodyOperation type representing an SDK body. It must be created with
+	// StartSDKBodyOperation() and finished with its Finish() method.
+	SDKBodyOperation struct {
+		dyngo.Operation
+		events json.RawMessage
```

Comment:
This is unused and unecessary. It is in the HTTP operation in order to aggregate the list of events happening during the HTTP handler lifetime, including those that would happen in the body operation.
 
```suggestion
```